pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/extensions/channel/ServiceUnitStateChannelImpl.java

```diff
@@ -1381,8 +1400,10 @@ private void overrideOwnership(String serviceUnit, ServiceUnitStateData orphanDa
 
     private void waitForCleanups(String broker, boolean excludeSystemTopics, int maxWaitTimeInMillis) {
         long started = System.currentTimeMillis();
```

Comment:
I think we can skip this waiting logic if the broker to cleanup is the last broker. 

I think this waiting logic is optional, and the intention is to give some time to the cluster to converge cleaned ownership view, if ever. 

However, because of the nature of the current ownership changes, there might be a chance that the ownership view might not converge within the timeout -- eventually the leader should fix any orphan or too long inflight states, though).

If there is no broker waiting, we can just skip this waiting.

Reply:
>  if the broker to cleanup is the last broker.

I found sometimes the cleanup did not fail for the last broker but for the 2nd broker in `testCloseAfterLoadingBundles`.

```
2024-09-26T15:35:56,001 - INFO  - [main:BrokerService] - Disable broker in load manager completed in 0.081 seconds
2024-09-26T15:36:31,287 - INFO  - [main:BrokerService] - Disable broker in load manager completed in 35.099 seconds
2024-09-26T15:36:31,644 - INFO  - [main:BrokerService] - Disable broker in load manager completed in 0.214 seconds
2024-09-26T15:36:31,783 - INFO  - [main:ExtensibleLoadManagerCloseTest] - Brokers close time: [275, 35242, 354]
```

Let me look it deeper

Reply:
The cause when the 2nd broker failed to close is the issue 5 I mentioned in the latest PR description.